tests/ui/doc/doc_lazy_list.stderr

```diff
@@ -120,6 +132,198 @@ help: indent this line
 LL | ///   'protocol_descriptors': [
    |      +
 
+error: doc list item overindented
+  --> tests/ui/doc/doc_lazy_list.rs:58:5
+   |
+LL | ///      {
+   |     ^^^^^
+   |
+   = help: if this is supposed to be its own paragraph, add a blank line
+help: indent this line
+   |
+LL | ///   {
+   |     ++
+
+error: doc list item overindented
+  --> tests/ui/doc/doc_lazy_list.rs:59:5
+   |
+LL | ///          'protocol': 25,  # u64 Representation of ProtocolIdentifier::AVDTP
+   |     ^^^^^^^^^
+   |
+   = help: if this is supposed to be its own paragraph, add a blank line
+help: indent this line
+   |
+LL | ///   'protocol': 25,  # u64 Representation of ProtocolIdentifier::AVDTP
+   |     ++
+
+error: doc list item overindented
+  --> tests/ui/doc/doc_lazy_list.rs:60:5
+   |
+LL | ///          'params': [
+   |     ^^^^^^^^^
+   |
+   = help: if this is supposed to be its own paragraph, add a blank line
+help: indent this line
+   |
+LL | ///   'params': [
+   |     ++
+
+error: doc list item overindented
+  --> tests/ui/doc/doc_lazy_list.rs:61:5
+   |
+LL | ///              {
+   |     ^^^^^^^^^^^^^
+   |
+   = help: if this is supposed to be its own paragraph, add a blank line
+help: indent this line
+   |
+LL | ///   {
+   |     ++
+
+error: doc list item overindented
+  --> tests/ui/doc/doc_lazy_list.rs:62:5
+   |
+LL | ///                 'data': 0x0103  # to indicate 1.3
+   |     ^^^^^^^^^^^^^^^^
+   |
+   = help: if this is supposed to be its own paragraph, add a blank line
+help: indent this line
+   |
+LL | ///   'data': 0x0103  # to indicate 1.3
+   |     ++
+
+error: doc list item overindented
+  --> tests/ui/doc/doc_lazy_list.rs:63:5
+   |
+LL | ///              },
+   |     ^^^^^^^^^^^^^
+   |
+   = help: if this is supposed to be its own paragraph, add a blank line
+help: indent this line
+   |
+LL | ///   },
+   |     ++
+
+error: doc list item overindented
+  --> tests/ui/doc/doc_lazy_list.rs:64:5
+   |
+LL | ///              {
+   |     ^^^^^^^^^^^^^
+   |
+   = help: if this is supposed to be its own paragraph, add a blank line
+help: indent this line
```

Comment:
copy+paste error? Over-indenting and under-indenting both say this.

Reply:
Ah, yes. I'll change them for over-indenting cases. Thanks.